python/tests/lang/definitions/test_schema.py

```diff
@@ -0,0 +1,109 @@
+from unittest import TestCase
+
+from aac.lang.active_context_lifecycle_manager import get_active_context
+from aac.lang.definitions.schema import get_definition_schema, get_root_schema_definitions, get_schema_defined_fields
+from aac.spec import get_root_fields
+from tests.helpers.context import get_core_spec_context
+
```

Comment:
I'd like to group the `tests.helpers` imports together.

```suggestion

from tests.helpers.context import get_core_spec_context
```